src/firebaseDb.ts

```diff
@@ -31,6 +31,8 @@ const testFirestoreIsInitialized = async (newDbInstance: Firestore) => {
     }
 }
 
+// This is to prevent race conditions where parallel calls/tests/etc 
+// try to initialize the firebase instance at the same time
 let alreadyStartedInitialization = false
```

Comment:
Are you sure this is needed? I was under the impression that `initializeApp()` will throw an error if it's called more than once for the same application name. [source](https://stackoverflow.com/questions/75241561/where-does-firebase-getapps-length-check-come-from)

Reply:
It was throwing an error during the tests since they run in parallel.